File: atcoder/python/templates/make_division.py
def make_divisors(n: int) -> list:
    return_list = []
    for i in range(1, int(n**0.5)+1):
        if n % i == 0:
            return_list.append(i)
            if i != n // i:
                return_list.append(n//i)

    return return_list

print(make_divisors(10))
print(make_divisors(25))
print(make_divisors(31))
print(make_divisors(655))

File: atcoder/rust/templates/make_division.rs
fn make_divisors(n: &usize) -> Vec<usize> {
    let mut divisors = Vec::new();
    let mut i = 1 as usize;
    while i * i <= *n {
        if n % i == 0 {
            divisors.push(i);
            if i != (n / i) {
                divisors.push(n / i);
            }
        }
        i += 1;
    }
    return divisors
}

fn main() {
    println!("{:?}", make_divisors(&(10 as usize)));
    println!("{:?}", make_divisors(&(25 as usize)));
    println!("{:?}", make_divisors(&(31 as usize)));
    println!("{:?}", make_divisors(&(655 as usize)));
}
